src/platform/forms/components/review/PreSubmitSection.jsx

```diff
@@ -67,24 +106,43 @@ PreSubmitSection.propTypes = {
   }).isRequired,
   showPreSubmitError: PropTypes.bool,
   setPreSubmit: PropTypes.func.isRequired,
+  user: PropTypes.shape({
+    login: PropTypes.shape({
+      currentlyLoggedIn: PropTypes.bool,
+    }),
+  }),
+  showLoginModal: PropTypes.bool,
+  saveAndRedirectToReturnUrl: PropTypes.func,
+  toggleLoginModal: PropTypes.func,
+  // added by withRouter
+  location: PropTypes.shape({
+    pathname: PropTypes.string,
+  }),
 };
 
 const mapDispatchToProps = {
   setPreSubmit: setPreSubmitAction,
+  saveAndRedirectToReturnUrlAction,
```

Comment:
This may need to be:

```suggestion
  saveAndRedirectToReturnUrl: saveAndRedirectToReturnUrlAction,
  ```